src/datadog/tracer_config.cpp

```diff
@@ -366,6 +366,7 @@ Expected<FinalizedTracerConfig> finalize_config(const TracerConfig &config,
   }
 
   result.runtime_id = config.runtime_id;
+  result.integration_name = config.integration_name;
```

Comment:
Where does "integration _version_" come from? Here we are in the land of names...

- Tracing library is dd-trace-cpp and its release version.
- Service is designated by the end-user (but the integration chooses a default). `config.defaults.version` is the version of that service, so also determined by the end-user.
- Integration is determined by who is offering a trace-compatible product to the end-user. So that might be the maintainers of Envoy, Istio, nginx-datadog, Kong, etc. The integration has a version separate from the other two notions of version. Let's call it the "integration version."

Where does "integration version" get configured? I see that it ends up in `class Tracer`.